Epsilon.Host.WebApi/Controllers/LearningController.cs

```diff
@@ -37,4 +37,18 @@ public async Task<ActionResult<LearningDomain>> GetDomain(string name)
 
         return Ok(domain);
     }
+    
+    
+    //TODO filter on tenetID and DomainID
+    [HttpGet("domain/{tenetId}/outcomes")]
```

Comment:
tenet should be tenant, also we should prevent commiting TODO stuff and instead move it to a separate PR. to keep the repo clean.